docs/overview.rst

```diff
@@ -0,0 +1,213 @@
+Project                      Kinto   Parse   Firebase  CouchDB  Remote-Storage
+---------------------------  ------  ------  --------  -------  --------------
+Fine-grained permissions     ✔       ✔       ✔
+Easy query mechanism         ✔       ✔       ✔         [#]_     [#]_
+Conflict resolution          ✔       ✔       ✔         ✔        ✔ [#]_
+Validation                   ✔       ✔       ✔         ✔
+Revision history                                       ✔
+File storage                         ✔                 ✔        ✔
+Batch/bulk operations        ✔       ✔                 ✔
+Changes stream               [#]_    ✔       ✔         ✔
+Pluggable authentication     ✔                         ✔        [#]_
+Pluggable storage / cache    ✔                                  ✔
+Self-hostable                ✔                         ✔        ✔
+Decentralised discovery      [#]_                               ✔
+Open source                  ✔                         ✔        ✔
+Language                     Python                    Erlang   Node.js [#]_
+===========================  ======  ======  ========  =======  ==============
+
+.. [#] CouchDB uses Map/Reduce as a query mechanism, which isn't easy to
+       understand for newcomers.
+.. [#] Remote Storage allows "ls" on a folder, but items are not sorted or
+       paginated.
+.. [#] Kinto uses the same mechanisms as Remote storage for conflict handling.
+.. [#] Notifications support is currently in the work.
+.. [#] Remote Storage supports OAuth2.0 implicit grant flow.
+.. [#] Support for decentralised discovery
+       `is planned <https://github.com/Kinto/kinto/issues/125>`_ but not
+       implemented yet.
+.. [#] Remote Storage doesn't define any default implementation (as it is
+       a procol) but makes it easy to start with JavaScript and Node.js.
+
+You can also read `a longer explanation of our choices and motivations behind the
+creation of Kinto <http://www.servicedenuages.fr/en/generic-storage-ecosystem>`_
+on our blog.
+
+
+.. _FAQ:
+
+FAQ
+===
+
+- How does Kinto compares to CouchDB / Remote Storage?
+
+    Before starting to create yet another data storage service, we had a long
+    look to the existing solutions, to see if that would make sense to extend
+    the community effort rather than re-inventing the wheel.
+
+    It appeared that solutions we looked at weren't solving the problems we had,
+    especially regarding fine-grained permissions.
+
+    To see how Kinto compares to these solutions,
+    read :ref:`the comparison table <comparison>`.
+
+- Can I encrypt my data?
+
+    Kinto server stores any data you pass to it, be it encrypted or not.
+    We make it easy to use encryption in our Kinto.js client
+    `using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
+
+- Is there a package for my Operating System?
+
+    Not at the moment. We want to make it very easy to integrate with existing
```

Comment:
Consider:
> Not yet! That said, we want to make Kinto as easy as possible to install and use, so this is high on our list of priorities.

Be careful when making promises like this, though. If you say "it's a top priority", and it sits for a year, then it's obviously not true. It might be safer (and more interesting) to invite the community to help out on this one.
Consider:
> No, but it's a great idea. Packaging is hard and we're a small team, so if you'd like to help us out by maintaining packages for your favourite OS, we'd be delighted to collaborate with you!